### DISCUSSION TITLE
ogaki mini maker faire 2024 展示作品`募集`

### DISCUSSION THREAD

**Discussion by gurezo** [chirimen-oh/meeting#37] [General]
## 目的

- [Ogaki Mini Maker Faire 2024](https://makezine.jp/blog/2024/03/ommf2024_announce.html) に当選しました。
  - 2024.11.23(土)~24(日)
- 上記イベントに展示する作品を募集します。

### 対象

- CHIRIMEN コミュニティの slack に参加している人対象

### 提供条件
- 事前に @gurezo or @tadfmac に作品を送って下さい
- 2024.11.15~~22~~(金)迄に送付して下さい（送付住所は、DM します。
- 組み立手順や注意事項を記載お願いします。（リポジトリ等の記載でも可
- 当日現地の持ち込みでも可又は参加可能上限2人まで

## ゴール

- [Ogaki Mini Maker Faire 2024](https://makezine.jp/blog/2024/03/ommf2024_announce.html) で、提供作品を展示します。

**Comment by tadfmac** [comment]
> 2024.11.22(金)迄に送付して下さい（送付住所は、DM します。    

これ、現地送付じゃないとムリじゃね？
@gurezo さんか私に預ける場合、たぶん、11/15くらいまで。と思います。11/16-17 で荷造り予定なので、それまでに。

私は11/22仕事お休みいただいて朝からゆっくり（鈍行で）移動予定。経費節約のため！

あと、CHIRIMENブース確か椅子余分に申し込んでるので、当日ブースで身体ごと出展する方も募集です。（大垣、10年くらい前のMFTくらいの感覚の規模で周りのブースの変態さんたちと交流する意味でもかなりおススメです。時間あれば！）

@gurezo. さん
私も CHIRIMEN向けの小さな展示を考えています。（幅2-30cmもあれば十分。スマホ＋小さなガジェット＋手作りダンボール小型スピーカー）
これは、自分のブースの荷物と一緒に送るorハンドキャリーして当日CHIRIMENブースに設置しにいく予定だけど、予め他の作品と一緒に並べてレイアウト考えたいぞ！という場合は教えてください。事前に @gurezo さん宛に送付も検討します。

**Reply by gurezo** [reply]
@tadfmac 
> これ、現地送付じゃないとムリじゃね？

時間の読みが甘かったですね。。。日付を訂正しました。

**Comment by gurezo** [comment]
2024 Ogaki mini maker faire は終了したので、close します。
